test/shared/utils.ts

```diff
@@ -21,3 +21,14 @@ export const getRoleErrorMsg = (
 ): string => {
   return `AccessControl: account ${account.address.toLowerCase()} is missing role ${role}`;
 };
+
+export const approxWholeTokens = (
```

Comment:
what does this function do?

Reply:
It divides the expected & actual number of tokens by 1e18 (or whatever decimals that tokens have) and then checks if they are equal or not. So this is basically ignoring the decimal difference and making sure the whole numbers of the tokens are equal.

Reply:
in that case, what do you think of taking a different approach and instead of this:

```
expect(approxWholeTokens(actualInvestedAssets, EXPECTED_INVESTED_ASSSETS))
        .to.be.true;
```

We could do something more explicit, like this:

```
expect(removeDecimals(actualInvestedAssets))
        .to.eq(removeDecimals(EXPECTED_INVESTED_ASSSETS));
```

I think this makes it more explicit about what's happening behind the scenes when reading the test.

Reply:
yes I agree. I will make the change.

Reply:
Fixed Sir